packages/lit-element/src/polyfill-support.ts

```diff
@@ -48,12 +48,14 @@ interface PatchableLitElement extends HTMLElement {
 }: {
   LitElement: PatchableLitElement;
 }) => {
+  const extraGlobals = window as unknown as LitExtraGlobals;
```

Comment:
Does this fully inline for a 0 byte change?

Reply:
I'm not sure. If it doesn't, would you prefer that each use is individually casted?

Reply:
The change is not fully inlined. Doing this seems to have made a couple of these files a bit smaller because `window` is now being aliased to a single letter variable in a few places:

![Screen Shot 2021-05-18 at 18 45 21](https://user-images.githubusercontent.com/406614/118744569-698d0c80-b809-11eb-9c2c-c69b2872c13f.png)

(I couldn't think of a better way to show a character diff besides a screenshot...)

What do you typically do to compare sizes across changes?

Reply:
We have a checksize script for that. Looks like in the core files most references here are guarded in dev mode checks, so we're ok. Smaller is better too :)